Reorganize "Update and migration" (#2519)
ibexa/documentation-developer: docs/update_and_migration/from_4.6/update_from_4.6.md

---
description: Update your installation to the latest v4.6 version from an earlier v4.6 version.
---

# Update from v4.6.x to v4.6.latest

## Update the application

Note which version you actually have before starting.

First, run:

=== "[[= product_name_headless =]]"

    ``` bash
    composer require ibexa/headless:[[= latest_tag_4_6 =]] --with-all-dependencies --no-scripts
    composer recipes:install ibexa/headless --force -v
    ```
=== "[[= product_name_exp =]]"

    ``` bash
    composer require ibexa/experience:[[= latest_tag_4_6 =]] --with-all-dependencies --no-scripts
    composer recipes:install ibexa/experience --force -v
    ```
=== "[[= product_name_com =]]"

    ``` bash
    composer require ibexa/commerce:[[= latest_tag_4_6 =]] --with-all-dependencies --no-scripts
    composer recipes:install ibexa/commerce --force -v
    ```

> Then execute the instructions below from the version you're upgrading from (or the closest superior) to the end of this page.

Reviewer: ```suggestion
Then execute the instructions below starting from the version you're upgrading from.
```
Can we add the missing versions with a short "No additional steps needed" section, like we have for 4.6.9? To make it clearer
